routes/index.js

```diff
@@ -281,15 +281,45 @@ router.get('/search', function(req, res, next) {
     var langFilter = req.body.langFilter
     var totalSongsDisplayed = req.body.totalSongsDisplayed
     var songs2d = []
-    var findOriginalSong = function(done) {
+
+    function findSongsLoggedIn(done) {
```

Comment:
because i started to like function declaration because of the hoisting behaviour